devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy

```diff
@@ -47,63 +47,63 @@ metadata {
         attribute "lastOpened", "String"
```

Comment:
This attribute should be changed to "lastWet"
Sorry - I also missed this in yesterday's push!